README.md

```diff
@@ -177,7 +177,7 @@ More examples can be found in the [examples directory](https://github.com/zerodh
 ## Documentation
 
 - [KiteConnect API documentation](https://kite.trade/docs/connect/v3/)
-- [CPPKiteConnect reference documentation](https://kite.trade/docs/cppkiteconnect/index.htmll#autotoc_md14)
+- [CPPKiteConnect reference documentation](https://kite.trade/docs/cppkiteconnect/)
```

Comment:
This should be `https://kite.trade/docs/cppkiteconnect/index.html#autotoc_md14`.

Content on the index page above `Documentation` is exact duplicate of README. Only `Documentation` section has information that README doesn't already have. This link will be clicked by users who have already seen the README. `#autotoc_md14` skips to the `Documentation` section so user doesn't have to go through duplicate content.

Reply:
I have added a permanent id to it as linking to autotoc_id would break eventually. Updated the link accordingly.

(Aside: I think there is a bit of duplication here with the mainpage.md and README.md; it seems with every update for the README both files would need to be updated which seems kind of redundant. Unsure if that can be removed easily but would be super useful to remove long term consistency between these files)